packages/cfd/src/Containers/props.types.ts

```diff
@@ -7,7 +7,7 @@ import {
     VerifyEmailResponse,
 } from '@deriv/api-types';
 import { FormikHelpers as FormikActions } from 'formik';
-import { TCFDPasswordFormValues } from './cfd-password-modal';
+import { TCFDPasswordFormValues } from './cfd-password-modal/cfd-password-modal';
```

Comment:
@markwylde-deriv shouldn't this imported from './types'?

Reply:
Ahh awesome. Fixed in e9c90ccdf92d20bdf7828dc74304712da87c0983.